sylpheed.txt

```diff
@@ -0,0 +1,20 @@
+[Section]
+Name = Sylpheed
+Version = 3.7.0
+License = GPL
+Description = Lightweight and user-friendly e-mail client.
+Size = 7.85 MiB
```

Comment:
If `SizeBytes` is specified, this one is not needed anymore :)
Rapps will automatically format the `Size` according to system settings.

Reply:
Removed.
But I wonder: if `Size` and `SizeBytes` are different, will it display the same correct `Size` value? Or it will be just converted from B into MB? E. g. 8.23 MB instead of 7.85.